### ISSUE TITLE
Add "bulk" processing in command-line usage

### ISSUE THREAD

**Issue by joskfg** [mawrkus/joey-the-differ#7]
# Description 
Currently the command line allow us to do the diff between two different files, but when you use a tool like that, usually it is done for several files.

I propose to be able to pass as a source and target directories instead of files, to get all the files inside as files to be diffed. The matching between source and target would be done using the filename.

## Usage example:
```
$ ls ./source
file-1.json file2,json

$ ls ./target
file-1.json file2,json

$ joey-the-differ -s ./source -t ./target -c ./config.js
<output>
```

## Other improvements
Some improvements can be done for this basic usage. 

### Parallelizing
To parallelize passing some parameter to tell how many processors use.
```
$ joey-the-differ -s ./source -t ./target -c ./config.js -p4
```

### Output in folder
To store all the results in an ordered way in some other folder, like for example:
```
$ ls ./source
file-1.json file2,json

$ ls ./target
file-1.json file2,json

$ ls ./results


$ joey-the-differ -s ./source -t ./target -c ./config.js -o ./results

$ ls ./results
file-1.json file2,json
```

Thanks for your work!

**Comment by mawrkus** [comment]
WiP: https://github.com/mawrkus/joey-the-differ/pull/9

**Comment by joskfg** [comment]
WDYT about the pros/cons of the output in a single file or multiple files?

**Comment by mawrkus** [comment]
haven't thought about it yet. It could be an option, I'll work on a 1st version and see the benefits/use cases.

**Comment by mawrkus** [comment]
I've implemented both options: single output files or many output files. Have a look at the PR and tell me what you think. The README is also updated.

**Comment by mawrkus** [comment]
#9 merged and v1.5.0 published.
